docs/RELEASE.md

```diff
@@ -0,0 +1,28 @@
+# Release process
+
+* Ensure docs/CHANGELOG.md contains a one-line summary of each significant
+  change since the prior release
+* Update setup.py and tuf/__init__.py to the new version number vA.B.C
+* Test packaging, uploading to Test PyPI and installing from a virtual environment
+  * remove existing dist build dirs
+  * create source dist `python setup.py sdist`
+  * create wheel (with 2 and 3 support) `python setup.py bdist_wheel --universal`
+  * Sign the dists `gpg --detach-sign -a dist/tuf-vA.B.C.tar.gz`
+  * Upload to test PyPI `twine upload --repository testpypi dist/*`
```

Comment:
Can't currently perform this step, as @vladimir-v-diaz is the owner of that package on test.pypi.org